plat/qemu/qemu_console.c

```diff
@@ -0,0 +1,37 @@
+/*
+ * Copyright (c) 2015-2018, ARM Limited and Contributors. All rights reserved.
+ *
+ * SPDX-License-Identifier: BSD-3-Clause
+ */
+#include <console.h>
+#include <pl011.h>
+#include <platform_def.h>
+
+static console_pl011_t console;
+static console_pl011_t crash_console;
+
+void qemu_console_init(void)
+{
+#if MULTI_CONSOLE_API
```

Comment:
You shouldn't really need the #if here or in any of the other plat/qemu/* files. If you're defining MULTI_CONSOLE_API to be 1 in the platform.mk, it should be okay to just assume that it's 1 in all corresponding code.